internal/oci/runtime_oci.go

```diff
@@ -525,14 +532,21 @@ func (r *runtimeOCI) ExecSyncContainer(ctx context.Context, c *Container, comman
 	args := []string{
 		"-c", c.ID(),
 		"-n", c.name,
-		"-r", r.handler.RuntimePath,
 		"-p", pidFile,
 		"-e",
 		"-l", logPath,
 		"--socket-dir-path", r.config.ContainerAttachSocketDir,
 		"--log-level", logrus.GetLevel().String(),
 	}
 
+	if archRuntimePath, ok := r.handler.ArchRuntimePaths[c.GetArch()]; ok {
```

Comment:
the equivalent piece in podman uses both os and platform https://github.com/containers/common/blob/cac40138f7e3c2b29ca32e64348535516bf6aa51/pkg/config/config.go#L695

@flouthoc what are some cases we need to include os as well as arch here? like does the runtime potentially differ based on OS between different wasm arches?

Reply:
@haircommander There are some defaults here https://github.com/containers/common/pull/1225, so i think cases can be pretty much `wasm/wasi*`

Reply:
I am wary of supporting wildcards if we don't need to. I suppose we could just have this be os to path. Do you see a world where we'd need a different runtime for a different arch @flouthoc ?

Reply:
My bad no I did not mean wildercard it was just to mention all three formats 
```
"wasi/wasm
"wasi/wasm32
"wasi/wasm64
```

Reply:
I am leaning towards including os and arch similar to podman to have the two implementations line up

Reply:
works for me

Reply:
Golang recently added something for wasm for OS and Arch. Let's look at that.

Reply:
Yes, Go added `OS=wasip1` for `wasi_snapshot_preview1` so maybe also add `wasip1/wasm`